content/posts/newsletter-017/index.md

```diff
@@ -61,6 +57,32 @@ _Discussions:
 If needed, a section can be split into subsections with a "------" delimiter.
 -->
 
+## [Rust GameDev Podcast #4][podcast-4]
```

Comment:
Tangentally related - we should call out the next meetup in the next newsletter :)

Reply:
@ozkriff I'll quickly do a writeup that you can put wherever if you like

Reply:
```
The Rust Gamedev meetup will happen on the second Saturday of every month at 16:00 GMT+0. The meetup will be an opportunity to chat with others from the Rust game development community and see what people are working on. The meetup will take place on the Rust Gamedev Discord server:

https://discord.gg/yNtPTb2
```

![meetup](https://user-images.githubusercontent.com/14791619/104633256-e302cb80-566c-11eb-8ebd-79113a93bb99.jpg)

Reply:
added